lib/usage.js

```diff
@@ -28,17 +28,22 @@ module.exports = function (yargs, y18n) {
     return self
   }
 
+  var outputFailure = true
```

Comment:
I'd be tempted to start the variable in a false state, just a personal preference when I'm using a variable as a flag. perhaps: `failureOutput` = `false`, and set it to `true` once you spit out an error -- just a minor nit feel free to ignore.